budgetportal/tests/test_infrastructure_projects.py

```diff
@@ -281,29 +272,25 @@ def setUp(self):
     @mock.patch('budgetportal.models.InfrastructureProjectPart.get_dataset', return_value=None)
     def test_missing_dataset_returns_404(self, mock_dataset):
         c = Client()
-        response = c.get('/infrastructure-projects/{}.yaml'.format(self.project.project_slug))
+        response = c.get('/infrastructure-projects/{}'.format(self.project.project_slug))
         self.assertEqual(response.status_code, 404)
 
     @mock.patch('budgetportal.models.InfrastructureProjectPart.get_dataset', return_value=MockDataset())
     @mock.patch('requests.get', side_effect=mocked_requests_get)
     def test_success_with_projects(self, mock_dataset, mock_get):
         """ Test that it exists and that the correct years are linked. """
         c = Client()
-        response = c.get('/infrastructure-projects/{}.yaml'.format(self.project.project_slug))
-        content = yaml.load(response.content)['projects'][0]
-
-        self.assertEqual(content['dataset_url'], 'fake path')
-        self.assertEqual(content['description'], self.project.project_description)
-        self.assertEqual(content['infrastructure_type'], self.project.infrastructure_type)
-        self.assertEqual(len(content['coordinates']), 0)
-        self.assertEqual(len(content['expenditure']), 7)
-        for item in self.project.build_complete_expenditure():
-            self.assertIn(item, content['expenditure'])
-        self.assertEqual(content['nature_of_investment'], self.project.nature_of_investment)
-        self.assertEqual(content['page_title'], '{} - vulekamali'.format(self.project.project_name))
-        self.assertEqual(content['projected_budget'], self.project.calculate_projected_expenditure())
-        self.assertEqual(len(content['provinces']), 1)
-        self.assertIn('', content['provinces'][0])
-        self.assertEqual(content['slug'], '/infrastructure-projects/{}'.format(self.project.project_slug))
-        self.assertEqual(content['stage'], self.project.current_project_stage)
-        self.assertEqual(content['total_budget'], self.project.total_project_cost)
+        response = c.get('/infrastructure-projects/{}'.format(
+            self.project.project_slug))
+        content = response.content
+
+        self.assertTrue(content.find('fake path'))
```

Comment:
please use `assertContains` rather than `assertTrue(content.find` because assertContains is logically equivalent but gives clearer explanation what went wrong when it fails

https://docs.djangoproject.com/en/2.2/topics/testing/tools/#django.test.SimpleTestCase.assertContains